avocado/core/sysinfo.py

```diff
@@ -412,6 +412,20 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
             log.debug('File %s does not exist.', commands_file)
             self.commands = []
 
+        fail_commands_file = settings.get_value('sysinfo.collectibles',
+                                                'fail_commands',
+                                                key_type='path',
+                                                default='')
```

Comment:
We have introduced the new `core.future.settings` module to handle configuration options. The current settings will be deprecated soon.

While accepting this current setting is ok, we should try to migrate to the new model, where default values are no more defined when "getting" (`config.get()`) values, but when "registering" (`settings.register_option()`) values.

Please, take a look into other examples of `core.future.settings` in the source code, and let me know if you have any questions.

Reply:
@beraldoleal Looks like we are still in the older method in case of other sysinfo, do you want me to migrate existing ones also to new register method ? Or can that be taken as a separate patch ?